src/Sentry/Laravel/Features/StorageIntegration.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+namespace Sentry\Laravel\Features;
+
+use Illuminate\Contracts\Filesystem\Cloud as CloudFilesystem;
+use Illuminate\Contracts\Filesystem\Filesystem;
+use Illuminate\Contracts\Foundation\Application;
+use Illuminate\Filesystem\FilesystemManager;
+use RuntimeException;
+use Sentry\Laravel\Tracing\Storage\TracingCloudFilesystem;
+use Sentry\Laravel\Tracing\Storage\TracingFilesystem;
+
+class StorageIntegration extends Feature
+{
+    private const FEATURE_KEY = 'storage';
+
+    private const STORAGE_DRIVER_NAME = 'sentry';
+
+    public function isApplicable(): bool
+    {
+        return $this->isBreadcrumbFeatureEnabled(self::FEATURE_KEY)
+               || $this->isTracingFeatureEnabled(self::FEATURE_KEY);
```

Comment:
```suggestion
        return $this->isBreadcrumbFeatureEnabled(self::FEATURE_KEY)
            || $this->isTracingFeatureEnabled(self::FEATURE_KEY);
```

Forgive my nitpick-ness, I just think it is beatiful how `return` and a standard indentation of 4 spaces + `||` have the same length.